### ISSUE TITLE
Localizations not working, even with forceLanguageLocalization

### ISSUE THREAD

**Issue by yemartin** [ArtSabintsev/Siren#336]
## If you're experiencing a problem integrating Siren into your app, please provide the following information when posting a new issue:

- **Are you using the [latest version](https://github.com/ArtSabintsev/Siren/releases) of Siren?**: Yes (5.4.0 as of now)
- **What is your app's Bundle ID?**: n/a
- **When was the latest version of your app published to the App Store?**: Not published yet
- **Is your app published in the US App Store? If not, what App Store is it published in?**: n/a
- **Does Siren work if you plugin your app's `BundleID` (and `countryCode`, if necessary) into the Example app?**: Yes and no, see below

## Related issue

Potentially related issue: https://github.com/ArtSabintsev/Siren/issues/67 but it provided no clue.

## Issue description

When running an app with Siren on a phone set to Japanese, the update alert showed up in English. Trying to set `forceLanguageLocalization` to `.japanese` (or other localizations) did not work either, the alert was still in English.

We ended up specifying strings manually, but this is not ideal: it does not extrapolate the app name and version number in the `alertMessage`.

## Example app

With the example app:
- The hyperCriticalRulesExample alert shows up in English by default, even though the device is set to Japanese.
- but `forceLanguageLocalization` does work and shows the alert in Japanese.

## Our code:
```swift
import Siren

struct UpdatesMonitor {
  static func configure() {
    let siren = Siren.shared
    // siren.presentationManager = PresentationManager(
    //   forceLanguageLocalization: .japanese
    // )
    // Unfortunately, the forceLanguageLocalization above had no effect, so:
    siren.presentationManager = PresentationManager(
      alertTitle: "アップデートのお知らせ",
      alertMessage: "新規バージョンがご利用になれます。\n今すぐアップデートしてください。",
      updateButtonTitle: "アップデート"
    )
    siren.rulesManager = RulesManager(
      globalRules: .critical,
      showAlertAfterCurrentVersionHasBeenReleasedForDays: 0
    )
    siren.wail()
  }
}
```

And we do `UpdatesMonitor.configure()` after `window?.makeKeyAndVisible()`.

**Comment by ArtSabintsev** [comment]
Thanks for the post.

1. Have you tried the sample app? Does the bug exist there?
2. I will look into this issue this evening or weekend on my end.

**Comment by yemartin** [comment]
> 1. Have you tried the sample app? Does the bug exist there?

Yes I did. With the sample app the behavior was:

- The `hyperCriticalRulesExample` alert shows up in English by default, even though the device is set to Japanese.
- but `forceLanguageLocalization` does work and shows the alert in Japanese.

**Comment by ArtSabintsev** [comment]
Alright, I need to think about this and poke in the code. I will get back to you this weekend.

**Comment by yemartin** [comment]
> Alright, I need to think about this and poke in the code. I will get back to you this weekend.

Thanks a lot! And let me know how I can help debug this.

**Comment by ArtSabintsev** [comment]
I think what's broken is that it may not detect the device language properly without it being forced. For that, I need more time to verify.

**Comment by yemartin** [comment]
Yes, in the sample app, `forceLanguageLocalization: .japanese` does work for me too. But in our real app, it does not, which I really don’t understand how that could be since the other options to the presenter do work...

**Comment by yemartin** [comment]
I can try to trim our app to a skeleton that I could share, and see if it still reproduces the issue. Would that help?

**Comment by ArtSabintsev** [comment]
Please do that! Yes, that would be ideal!

**Comment by yemartin** [comment]
OK I’ll try that then. It may take a little bit of time, I’ll write back here when it is ready.

**Comment by ArtSabintsev** [comment]
No rush at all - thank you!

**Comment by yemartin** [comment]
I have trimmed down our app to the bare minimum, and the bug is still happening. Here is the repo:

https://github.com/mediweb/siren_localization_bug

**Comment by ArtSabintsev** [comment]
Looking into this now. Thanks for the project.

**Comment by ArtSabintsev** [comment]
```
2020-06-10 06:17:28.881915-0400 SirenLocalizationBug[49143:5047538] [Application] Failed to instantiate the default view controller for UIMainStoryboardFile 'Main' - perhaps the designated entry point is not set?
```

I tried changing `Main` to `Main.storyboard` in the settings, which autocompletes. However, that file is empty.

Also, I am on Xcode 11.5.

**Comment by yemartin** [comment]
Yes, the app does not show anything on the Main storyboard: it starts on the TOS storyboard. And if Siren is triggered, it shows the alert there.

Can you try setting the bundle ID to a bundle ID of an app you control with a published version higher than 1.0.0, to force the Siren alert to appear?

**Comment by yemartin** [comment]
I have further trimmed down the app and made it display a dummy label, to prevent confusion. Please pull the latest commit and try again.

**Comment by ArtSabintsev** [comment]
I have to make other changes to the package manifest as well, so I will try to get it this weekend. Sorry it's taking a long time - lots going on at the moment inside and outside of work. If you want to take a stab an updating the manifest, I welcome PRs!

**Comment by ArtSabintsev** [comment]
It looks like this can only be fixed when Swift 5.3 is released, which should be later this year. I’m closing this issue until then. Either do what you’re doing now the to fix this issue, or use a different dependency manager.
